data/spells/scripts/party/enchant.lua

```diff
@@ -10,9 +10,9 @@ condition:setParameter(CONDITION_PARAM_STAT_MAGICPOINTS, 1)
 condition:setParameter(CONDITION_PARAM_BUFF_SPELL, true)
 
 local baseMana = 120
```

Comment:
But if I move it into ``onCastSpell()`` won't it be created every time the spell is cast?